templates/traefik.yaml

```diff
@@ -11,6 +11,7 @@ spec:
     version: 1.68.4
     values: |
       ---
+      replicas: 3
```

Comment:
two replicas should be enough. Did we face any specific issue to increase the replicas ?

Reply:
hang on, this is `wip` I'm looking at the potential autoscaling feature and will come back to this shortly.

Reply:
Alright :), roger that.